core/templates/dev/head/components/create_button/CreateActivityButtonDirective.js

```diff
@@ -48,7 +50,11 @@ oppia.directive('createActivityButton', [function() {
           $scope.creationInProgress = true;
 
           if (!GLOBALS.can_create_collections) {
-            ExplorationCreationService.createNewExploration();
+            if (urlService.getPathname() !== '/dashboard') {
```

Comment:
This is to make sure that exploration creation goes through the editor page, right? If so, please add a comment.